l1-contracts/src/core/libraries/rollup/AttestationLib.sol

```diff
@@ -128,11 +127,14 @@ library AttestationLib {
   }
 
   /**
-   * Returns the addresses from the CommitteeAttestations, using the array of signers to populate where there are
-   * signatures.
-   * Indices with signatures will have a zero address.
+   * Recovers the committee from the addresses in the attestations and signers.
+   *
    * @param _attestations - The committee attestations
+   * @param _signers The addresses of the committee members that signed the attestations. Provided in order to not have
+   * to recover them from their attestations' signatures (and hence save gas). The addresses of the non-signing
+   * committee members are directly included in the attestations.
    * @param _length - The number of addresses to return, should match the number of committee members
+   * @return The addresses of the committee members.
```

Comment:
Got confused here why do we pass in the signers + this:

"Indices with signatures will have a zero address."

was a lie so updated the docs here.